src/transformers/models/phi/modeling_phi.py

```diff
@@ -242,18 +258,22 @@ def __init__(self, config: PhiConfig, layer_idx: Optional[int] = None):
                 f"hidden_size must be divisible by num_heads (got `hidden_size`: {self.hidden_size}"
                 f" and `num_heads`: {self.num_heads})."
             )
-        self.query_key_value = nn.Linear(self.hidden_size, 3 * self.hidden_size, bias=True)
+
+        # Copied from transformers.models.llama.modeling_llama.LlamaAttention.__init__ with llama->phi
```

Comment:
we can't use copied from here 🤗 
The attention layer is a tiny bit "new" so no need to use copied from here I think
```suggestion
```